TestCases/compliance-level-3/0110-feel-filters/0110-feel-filters.dmn

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="https://github.com/dmn-tck/tck" name="feel-filters" id="_feel-filters" xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:triso="http://www.trisotech.com/2015/triso/modeling" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <decision name="Decision1" id="d_Decision1">
+    <variable typeRef="boolean" name="Decision1"/>
+    <literalExpression>
+      <text>[ 1, 2, 3, 4 ][ item > 2 ] = [ 3, 4 ]</text>
+    </literalExpression>
+  </decision>
+  <decision name="Decision2" id="d_Decision2">
+    <variable typeRef="boolean" name="Decision2"/>
+    <literalExpression>
+      <text>[ { x: 1, y: 2}, {x:2, y:3} ][ x = 1 ] = [ { x: 1, y: 2 } ]</text>
```

Comment:
Not sure if we need the first two as they are already covered in https://github.com/dmn-tck/tck/blob/master/TestCases/compliance-level-3/0069-feel-list/0069-feel-list.dmn#L183

However, I couldn't find existing test cases for the rest of the expressions.

Reply:
I was just following along existing good practices to test the actual examples, in a single place. Happy to remove what has already been covered elsewhere.

However I'd suggest to introduce explict specs for filters, as we have for `path`: https://github.com/dmn-tck/tck/pull/540#issuecomment-1396727466